js/app.js

```diff
@@ -1,7 +1,9 @@
 import { firstOrDefaultValue, firstOrDefaultContent } from "./parsing.js";
 import { buildBlogItemsUrl, buildTaxonomiesUrl, fetchKontent } from "./kontent.js";
 
-const tag = getUrlParamater("tag");
+const baseUrl = new URL(location);
+baseUrl.search = "";
```

Comment:
Don't you need to clear hash too? https://developer.mozilla.org/en-US/docs/Web/API/URL#Properties

Reply:
Yes, you are right